servers/audio/audio_driver_dummy.cpp

```diff
@@ -54,6 +54,7 @@ Error AudioDriverDummy::init() {
 }
 
 void AudioDriverDummy::thread_func(void *p_udata) {
+	Thread::set_name("Audio dummy");
```

Comment:
This is the Dummy audio driver:

```suggestion
	Thread::set_name("Audio Dummy");
```